src/Extractor/Output.php

```diff
@@ -107,7 +107,8 @@ private function createReportRow(Result $reportDataItem, string $profileId): arr
         $dimensions = $this->formatResultKeys($reportDataItem->getDimensions());
 
         if (isset($dimensions['date'])) {
-            $dimensions['date'] = date('Y-m-d', strtotime($dimensions['date']));
+            $time = strtotime($dimensions['date']) ?: 0;
```

Comment:
ve striktním modu to tu neprošlo, pokud se posílal string v `$dimension['date']`